### PR TITLE
[ADD] Module website_calendar_block 8.0

### PR DESCRIPTION
**Pull request by astirpe**
This module adds a new block in the website editor.
The new block is based on Messaging -> Calendar.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Seems very interesting!

These blocks are technically called "snippets", so you should rename it to _website_calendar_snippet_

**Comment by oscarolar** [comment]
found this error on runbot:
error

**Comment by astirpe** [comment]
@pedrobaeza Thank you for the compliments!
@oscarolar Thank you! Git was ignoring the lib directory while committing. Should be ok now.

File: website_calendar_snippet/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Calendar Snippet
+================
+
+This module adds a new block in the website editor.
+The new block is based on Messaging -> Calendar. Use field 'Privacy'
+to determine its publicity.
```
Comment (rafaelbn) [inline website_calendar_snippet/README.rst:11]: Where is the field 'Privacy'?

**Comment by rafaelbn** [comment]
Review and tested! :+1: thanks for contributing.  Please take care of commnets (I didn't get any runbot error)

**Comment by andhit-r** [comment]
Tested on runot :+1:

File: website_calendar_snippet/README.rst (hunk 1 above)
Comment (andhit-r) [inline website_calendar_snippet/README.rst:11]: To change privacy setting you should:
1. Click the event
2. Click **Edit Event** button
3. Go to **Options** tab

website_calendar

Comment (pedrobaeza) [inline website_calendar_snippet/README.rst:11]: @astirpe, can you please add this information on the Configuration section?

**Comment by pedrobaeza** [comment]
Except the little comment, :+1:

**Comment by oscarolar** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Thanks for the contribution, @astirpe. Merged!

Do you or your company have Twitter account for mentioning you?

**Comment by astirpe** [comment]
Thanks to you all !!!
Yes, it would be great to mention the Company on twitter, please check this account: https://twitter.com/onestein_erp

The module was entirely developed by Dennis! Many thanks @tarteo !

**Comment by astirpe** [comment]
Oh, and... here is the twitter account of Dennis https://twitter.com/onestein_dennis

**Pull request merged**